### ISSUE TITLE
Add page titles

### ISSUE THREAD

**Issue by Razzeee** [xbmc/kodi-tv#32]

**Comment by pkscout** [comment]
Found this: https://www.gatsbyjs.com/docs/add-page-metadata/

That will work for all the dynamically generated pages.  I'll have to see about what to do about the markdown files. Hopefully we won't have to add that to every page though. I'm mostly just putting this comment here to document what I've found so far (in case I close the tab with that page in it by accident).

**Comment by Razzeee** [comment]
Yeah, I found that page too and was quiet confused what to do with the mdx pages :D

**Comment by pkscout** [comment]
Done with https://github.com/xbmc/kodi-tv/commit/9899d973574cfb71a794eecb8226659afef2228b. I wrote a little component that adds to the metadata header using Helmet and accepts "title" as a passed property.  So that each page can have it's own title, the template built pages have that component added to the template with the dynamic information needed for each page to get a unique title. For the mdx pages, I imported the component and passed a page title in.  The behavior mirrors the current site, so the main site has a title, and other pages prepend to that (so Kodi | Opensource Home Theather Software is the main title, and the about page is About Us | Kodi | Opensource Home Theather Software).
